.github/workflows/node.js.yml

```diff
@@ -0,0 +1,30 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ '**' ]
```

Comment:
also `dev`? no one wants their dev broken

Reply:
It was done since gh starts double run of action (one for push and one for pullrequst), will do at another way

Reply:
Now it's for pushes to any branch?

Reply:
yes, including pushes for pull requests (you can see 2 successful checks in this PR for the last commit)